Handbook - Writing guidelines reorg (#33259)
fleetdm/fleet: handbook/company/communications.md

### Writing style

> 

Author: ```suggestion

- [Writing style](#writing-style)
- [Writing types](#writing-types)
- [Editing and publishing](#editing-and-publishing)
- [Further inspiration](#further-inspiration)
- [Writing assistance](#writing-assistance)
- [Writing mechanics](#writing-mechanics)
- [Writing in Fleet-flavored Markdown](#writing-in-fleet-flavored-markdown)
```

### Writing style

> - [Writing style](#writing-style)

Reviewer: ```suggestion
- [Writing style](https://fleetdm.com/handbook/company/communication#writing-style)
```
With all the moving around that happens in the handbook, I always use the full urls

> - [Writing types](#writing-types)

Reviewer: ```suggestion
- [Writing types](https://fleetdm.com/handbook/company/communication#writing-types)
```
Same

> - [Editing and publishing](#editing-and-publishing)

Reviewer: ```suggestion
- [Editing and publishing](https://fleetdm.com/handbook/company/communication#editing-and-publishing)
```

> - [Further inspiration](#further-inspiration)

Reviewer: ```suggestion
```
Can we move the sentence down below and cut down on the headings? Combined content elsewhere

> - [Writing assistance](#writing-assistance)

Reviewer: ```suggestion
- [Writing assistance](https://fleetdm.com/handbook/company/communication#writing-assistance)
```

> - [Writing mechanics](#writing-mechanics)

Reviewer: ```suggestion
- [Writing mechanics](https://fleetdm.com/handbook/company/communication#writing-mechanics)
```

> - [Writing in Fleet-flavored Markdown](#writing-in-fleet-flavored-markdown)

Reviewer: ```suggestion
- [Writing in Fleet-flavored Markdown](https://fleetdm.com/handbook/company/communication#writing-in-fleet-flavored-markdown)
```

Fleet’s writing style is clear, simple, and welcoming. We use short sentences, plain English, and an active voice so anyone can follow along. Instead of sounding formal, we aim for approachable and easy to read.

We infuse the company’s [values](https://fleetdm.com/handbook/company#values) into everything we write. That means being transparent, straightforward, and respectful of the reader’s time.

We avoid "[puffery](https://www.linkedin.com/pulse/puffery-adam-frankl%3FtrackingId=SBVWxzqXTBm9qlO7Rw3ddw%253D%253D/?trackingId=SBVWxzqXTBm9qlO7Rw3ddw%3D%3D)". For engineers, replace hype with real data. For business readers, translate it into clear outcomes such as time saved or return on investment. Links are better than long explanations, since they keep content short and point people to more detail when they need it.

When in doubt, simplify. Read your draft, cut unnecessary words, and make it shorter. If something feels confusing, rewrite until it feels obvious.

> Our approach is informed by [Paul Graham's essays on writing simply](http://www.paulgraham.com/simply.html) and the clarity and optimism of [Mister Rogers](#what-would-mister-rogers-say) (see the [Further inspiration](#further-inspiration) section, below).

Reviewer: ```suggestion
Our approach is informed by [Paul Graham's essays on writing simply](http://www.paulgraham.com/simply.html) and the clarity and optimism of Mister Rogers. To see how tone can shift from formal or negative to simple and optimistic, [the "Mister Rogersing" example](https://fleetdm.com/handbook/company/communication#what-would-mister-rogers-say) is a practical illustration of how reframing can make complex or difficult ideas more approachable.
```

> 

Reviewer: ```suggestion
```
No need for the extra line

### Writing types

Different types of writing at Fleet have slightly different expectations. Keep the shared principles in mind (plain English, brevity, clarity), but adjust for the format.

> 

Reviewer: ```suggestion

```
Need one here, though.

#### Guides and tutorials

- Write in short sentences, imperative mood, and active voice.
  - Example: “Click Save.” not “The button should then be clicked.”
- Format as directional, step-by-step instructions, not narrative prose.
- Use **bold** text when referencing UI elements.
  - Example: “Navigate to **Hosts** and click **Add a host**.”
- Surface the simple, high-level steps first.
- Place advanced or technical details, including troubleshooting, in a separate section at the bottom.
- Keep it practical, avoid marketing fluff, superlatives, and unnecessary adjectives.

#### Announcements

- Lead with the news. Put the key point in the first sentence.
- Keep it brief (two to four sentences).
- Use plain, factual language without fluff.
- Include a clear link or call to action if readers need to follow up.

> #### Articles

Reviewer: ```suggestion

#### Articles
```

- Keep the tone conversational and approachable, while still representing Fleet.
- Provide context or insight (the “why”), not just the “what.”
- Avoid jargon unless you explain it.
- Stay aligned with Fleet’s values and overall voice.

> #### Website copy

Reviewer: ```suggestion

#### Website copy
```

- Keep it simple: short sentences, plain English, imperative mood.
- Avoid fluff, filler, or jargon.
- Emphasize reader outcomes. Show what someone can do with Fleet, not just what Fleet is.
  - Example: _“Manage all your devices in one place”_ instead of _“Fleet is the leading platform for device management.”_
- Use headings and subheadings to make scanning easy.
- Keep calls to action direct and specific 
  - Example: _“Try Fleet”_ instead of _“Learn more about our amazing platform.”_

> ### Editing and publishing

Reviewer: ```suggestion

### Editing and publishing
```

Follow these steps before merging a change:

- Avoid unnecessary changes to headings, since this can break handbook links shared in other places.
- Link instead of duplicating content. If a concept exists elsewhere, link to it rather than restating it.
- Review your pull request carefully. Read the diff line by line until it looks intentional.
- Check preview mode in GitHub to make sure formatting renders correctly.
- Look for and remove any unintentional changes in your diff.

### Further inspiration

> To see how tone can shift from formal or negative to simple and optimistic, The "The Mister Rogersing" example below is a practical illustration of how reframing can make complex or difficult ideas more approachable.

Reviewer: Can we move the sentence down below and cut down on the headings?

> ### Further inspiration
> 
> To see how tone can shift from formal or negative to simple and optimistic, The "The Mister Rogersing" example below is a practical illustration of how reframing can make complex or difficult ideas more approachable.

Reviewer: ```suggestion
```

By Mister Rogersing our writing, we can encourage our readers to succeed by emphasizing optimism. You might not be able to apply all of these steps every time. That’s fine. Think of these as guidelines to help you simplify complex topics.

### Writing assistance

#### Grammarly

All of our writers and editors have access to Grammarly, which comes with a handy set of tools, including:
- **Style guide**, which helps us write consistently in the style of Fleet.
- **Brand tones** to keep the tone of our messaging consistent with just the right amount of confidence, optimism, and joy.
- **Snippets** to turn commonly used phrases, sentences, and paragraphs (such as calls to action, thank you messages, etc.) into consistent, reusable snippets to save time.

#### Generative AI

Collaborating with AI can be helpful for outlines, rewrites, and drafts. But it’s not a substitute for judgment. You (a human) are responsible for accuracy, tone, and format. 
- Don’t paste sensitive data.
- Always fact-check names, versions, dates, and links.
- Question everything you don't understand and strip out anything fabricated.

> - Make sure the tone sounds like Fleet (plain English, short sentences, active voice) and watch for common AI habits like em dashes, over-bolding, clichés, or passive voice.

Reviewer: ```suggestion
- Make sure the tone sounds like Fleet (plain English, short sentences, active voice) and watch for common AI habits like the overuse of em dashes, over-bolding, or passive voice.
```
